### ISSUE TITLE
this.editorBuffer.getLastRow is not a function

### ISSUE THREAD

**Issue by aditiapratama** [bigfive/atom-sublime-select#97]
[Enter steps to reproduce below:]

1. Open the file
2. Pressing the shortcut will cause the error (I've tried with many combination as well)

**Atom Version**: 1.9.0-dev-4da23fd
**System**: "Fedora release
**Thrown From**: [Sublime-Style-Column-Selection](https://github.com/bigfive/atom-sublime-select) package, v1.5.1

### Stack Trace

Uncaught TypeError: this.editorBuffer.getLastRow is not a function

```
At /home/aditia/.atom/packages/Sublime-Style-Column-Selection/lib/editor-handler.coffee:92

TypeError: this.editorBuffer.getLastRow is not a function
    at SublimeSelectEditorHandler.module.exports.SublimeSelectEditorHandler._screenPositionForMouseEvent (/home/aditia/.atom/packages/Sublime-Style-Column-Selection/lib/editor-handler.coffee:92:58)
    at SublimeSelectEditorHandler.module.exports.SublimeSelectEditorHandler.onMouseDown (/home/aditia/.atom/packages/Sublime-Style-Column-Selection/lib/editor-handler.coffee:41:27)
    at atom-text-editor.<anonymous> (/home/aditia/.atom/packages/Sublime-Style-Column-Selection/lib/editor-handler.coffee:1:1)
```

### Commands

```
     -6:06.6.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -6:05.9.0 vim-mode:move-right (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -6:05.5.0 vim-mode:move-left (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
     -6:03.6.0 vim-mode:activate-insert-mode (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
 48x -6:03.2.0 core:select-right (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -5:56 core:undo (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -5:54.4.0 core:move-right (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -5:53.6.0 core:backspace (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
 21x -5:52.4.0 editor:move-to-beginning-of-word (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
  4x -5:48.2.0 core:move-left (atom-text-editor.editor.vim-mode.is-focused.insert-mode)
     -5:26.5.0 core:select-all (atom-text-editor.editor.mini.is-focused)
  2x -5:23.5.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -5:19.7.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:33.7.0 core:select-all (atom-text-editor.editor.vim-mode.insert-mode.is-focused)
     -0:27.8.0 core:backspace (atom-text-editor.editor.vim-mode.insert-mode.is-focused)
  2x -0:23.4.0 core:undo (atom-text-editor.editor.vim-mode.insert-mode.is-focused)
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "automaticallyUpdate": false,
    "disabledPackages": [
      "w3c-validation",
      "aligner-php",
      "autocomplete-php",
      "git-history",
      "expose",
      "tree-view-git-branch"
    ],
    "packagesWithKeymapsDisabled": [],
    "themes": [
      "nucleus-dark-ui",
      "atom-material-syntax-dark"
    ]
  },
  "Sublime-Style-Column-Selection": {}
}
```

### Installed Packages

```coffee
# User
.bin, vundefined (inactive)
advanced-open-file, v0.16.2 (inactive)
aligner-php, v1.0.0 (inactive)
atom-beautify, v0.29.7 (inactive)
atom-css-unit-converter, v1.1.1 (inactive)
atom-dark-fusion-syntax, v1.1.0 (inactive)
atom-material-syntax-dark, v0.2.5 (active)
atom-material-syntax-light, v0.4.5 (inactive)
atom-material-ui, v1.2.11 (inactive)
atom-monokai, v0.10.2 (inactive)
atom-typescript, v8.9.1 (active)
autocomplete-emojis, v2.3.0 (active)
autocomplete-paths, v1.0.2 (active)
autocomplete-php, v0.3.7 (inactive)
cake-syntax, v0.5.0 (inactive)
color-picker, v2.1.1 (active)
eclectic-syntax, v0.2.1 (inactive)
emmet, v2.4.3 (active)
expose, v0.11.1 (inactive)
file-icons, v1.7.9 (active)
firewatch-syntax, v1.0.1 (inactive)
fizzy, v0.16.0 (inactive)
font-viewer, v0.3.1 (active)
fonts, v1.2.0 (active)
genesis-ui, v0.5.0 (inactive)
git-blame, v0.4.8 (inactive)
git-control, v0.7.1 (inactive)
git-diff-details, v1.3.0 (active)
git-history, v3.2.0 (inactive)
git-log, v0.4.1 (active)
git-plus, v5.13.4 (inactive)
git-projects, v1.17.0 (inactive)
git-time-machine, v1.5.2 (inactive)
gruvbox-syntax, v2.3.1 (inactive)
linter, v1.11.4 (active)
merge-conflicts, v1.4.2 (active)
minimap, v4.23.3 (active)
monokai, v0.18.0 (inactive)
monokai-seti, v0.7.0 (inactive)
multi-cursor, v2.1.3 (active)
native-ui, v0.17.0 (inactive)
neon-selection, v0.3.2 (active)
nucleus-dark-ui, v0.7.20 (active)
open-in-browser, v0.4.7 (inactive)
open-terminal-here, v2.1.4 (inactive)
pigments, v0.26.0 (active)
project-manager, v2.9.7 (active)
project-plus, v0.8.2 (active)
Remote-FTP, v0.8.3 (active)
seti-syntax, v1.0.1 (inactive)
seti-ui, v1.0.4 (inactive)
sort-lines, v0.14.0 (inactive)
split-diff, v0.7.5 (inactive)
state-of-nature-syntax, v0.1.0 (inactive)
Sublime-Style-Column-Selection, v1.5.1 (active)
svg-preview, v0.7.4 (active)
sync-settings, v0.7.2 (active)
tag, v0.4.0 (active)
tango-syntax, v0.4.1 (inactive)
tool-bar, v0.4.0 (active)
tool-bar-almighty, v0.6.1 (active)
tree-view-git-branch, v0.1.2 (inactive)
tree-view-git-status, v0.2.3 (active)
vim-mode, v0.65.0 (active)
w3c-validation, v0.4.0 (inactive)
wordpress-api, v1.2.2 (active)
zip-folder, v3.2.0 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.51.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.43.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.3.1 (inactive)
one-light-ui, v1.3.1 (inactive)
one-dark-syntax, v1.2.0 (inactive)
one-light-syntax, v1.2.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.2 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.0 (active)
autocomplete-snippets, v1.10.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.0 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.0 (active)
command-palette, v0.38.0 (active)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.38.1 (active)
fuzzy-finder, v1.0.5 (active)
git-diff, v1.0.1 (active)
find-and-replace, v0.198.0 (active)
go-to-line, v0.30.0 (active)
grammar-selector, v0.48.1 (active)
image-view, v0.57.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.63.2 (active)
open-on-github, v1.1.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.236.0 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.2.6 (active)
styleguide, v0.45.2 (active)
symbols-view, v0.113.0 (active)
tabs, v0.93.2 (active)
timecop, v0.33.1 (active)
tree-view, v0.206.2 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.32.2 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.51.4 (active)
language-clojure, v0.20.0 (active)
language-coffee-script, v0.47.0 (active)
language-csharp, v0.12.1 (active)
language-css, v0.36.1 (active)
language-gfm, v0.86.0 (active)
language-git, v0.13.0 (active)
language-go, v0.42.0 (active)
language-html, v0.44.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.18.0 (active)
language-javascript, v0.110.0 (active)
language-json, v0.18.0 (active)
language-less, v0.29.3 (active)
language-make, v0.22.0 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.34.0 (active)
language-php, v0.37.0 (active)
language-property-list, v0.8.0 (active)
language-python, v0.43.1 (active)
language-ruby, v0.68.5 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.49.0 (active)
language-shellscript, v0.22.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.21.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.27.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.5 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by mairanteodoro** [comment]
Having the same issue here. Using Atom 1.9.0-beta0.

**Comment by nickgal** [comment]
I got it working by changing `@editorBuffer.getLastRow()` to `@editor.buffer.getLastRow()` on these lines:
https://github.com/bigfive/atom-sublime-select/blob/master/lib/editor-handler.coffee#L96-L97

**Comment by aditiapratama** [comment]
@nickgal that's great, I'll create patch for this

**Comment by mairanteodoro** [comment]
@nickgal That did the trick.

**Comment by ajoslin103** [comment]
+1 -- hope this fix is issued soon

**Comment by fbatista** [comment]
👍also run into this issue, thanks for the fix!

**Comment by ajoslin103** [comment]
that fix doesn't work for me in the 1.9.0-beta version, it just stops the error -- Max OS X
